cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1713,6 +1707,14 @@
       (reuslt VecWritableReg (vec_writable_clone dst))
       (_ Unit (emit (MInst.Select dst ty c x y))))
     (vec_writable_to_regs reuslt)))
+  
+;; paramter is "intcc compare_a compare_b rs1 rs2".
```

Comment:
On the one hand I'm tempted to suggest fixing "paramter" to "parameter" while we're here, and on the other hand I really like that this PR makes zero changes to anything it's moving. Hmm… I'll leave it up to you to decide :grin: